qa/workunits/fs/misc/subvolume.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh -x
+
+if test $(stat -c '%i' .) != 1; then
+	echo "cwd is not cephfs root"
+	exit 0
+fi
+
+expect_failure() {
+	if "$@"; then return 1; else return 0; fi
+}
+
+set -e
+
+mkdir subvol1
+mkdir subvol1/dir
+
+expect_failure setfattr -n ceph.dir.subvolume -v 1 subvol1/dir
```

Comment:
Why should this fail?

Reply:
subvolume is only allowed in root directory